examples/send_batch_with_jobs.py

```diff
@@ -0,0 +1,28 @@
+from pasqal_cloud import SDK
+from pasqal_cloud.device import EmulatorType
+
+project_id = (
+    "your_project_id"  # Replace this value by your project_id on the PASQAL platform.
+)
+username = "your_username"  # Replace this value by your username or email on the PASQAL platform.
+password = (
+    "your_password"  # Replace this value by your password on the PASQAL platform.
```

Comment:
`# Replace this value by ` -> `# Replace this value with`

Reply:
I will update this error in the others parts of the readme, thx for spotting it